on-prem-docs/builder-automate.md

```diff
@@ -95,6 +95,8 @@ To authenticate with Chef Automate, create a patch with the Chef Automate comman
     ```
 
 1. Edit `bldr.env`:
+    * You must have version chef/automate-builder-api-proxy/0.1.0/20230111051525 or higher of builder-api-proxy, you can check by viewing the hab package storage area on your system
```

Comment:
If they are using `builder-api-proxy` via the automate wrapped `chef/automate-builder-api-proxy` then they are using an integrated automate bundled builder which should have the same origin domain as automate and should not suffer the cross-origin issue. This becomes an issue when using a separate standalone on-prem builder.